examples/docqa/doc-chat-2.py

```diff
@@ -157,8 +157,8 @@ def main(
         assistant_mode=True,
         parsing=ParsingConfig(  # modify as needed
             splitter=Splitter.TOKENS,
-            chunk_size=1000,  # aim for this many tokens per chunk
-            overlap=100,  # overlap between chunks
+            chunk_size=500,  # aim for this many tokens per chunk
```

Comment:
is there any rule you are following when you change these values of `chunk_size` and `overlap`?

Reply:
no it's trial and error.